config/locales/defaults/download_upload/en.yml

```diff
@@ -5,6 +5,8 @@ en:
   download_csv: Download in CSV Format
   download_errors:
     unrecognized_format: Could not recognize %{format} format to download with
+  download_json: Download in JSON Format
+  download_tests: Download Test Results
```

Comment:
Instead of creating a new key, re-use the `download_the` key (see below), together with the model name for test results (see `config/locales/models/test_results`).